### ISSUE TITLE
Default messaging for receivers of 'whisper' command cannot be overridden.

### ISSUE THREAD

**Issue by awranthyr** [evennia/evennia#1899]
#### Brief summary of issue / Description of requested feature:
When implementing an overloaded version of the default 'whisper' command, the messaging issued to all characters on the receiving end is reverted to the default format.

#### Steps to reproduce the issue / Reasons for adding feature:

1. Create a custom CmdWhisper and overload the default version by adding it to your character command set.
2. Pass a custom message through msg_receivers in the call to the character object's at_say method.
3. Reload the server and observe that the customized messaging is not utilized on the receiving end of the command.

#### Error output / Expected result of feature
With custom messaging defined as:
```
msg_receivers = '|c{object} whispers,|n "{speech}"'
```

The characters on the receiving end actually receive:
```
Awranthyr whispers: "Test."
```

#### Extra information, such as Evennia revision/repo/branch, operating system and ideas for how to solve / implement:
Evennia 0.9.0, Main branch, revision 3abb2f3f

At a quick glance it seems the issue would be solved by removing the following line (line 1849 at the time of this writing):
```
            msg_receivers = '{object} whispers: "{speech}"'
```

... From this snippet of evennia/objects/objects.py:
```
        if kwargs.get("whisper", False):
            # whisper mode
            msg_type = 'whisper'
            msg_self = '{self} whisper to {all_receivers}, "{speech}"' if msg_self is True else msg_self
            msg_receivers = '{object} whispers: "{speech}"'
            msg_receivers = msg_receivers or '{object} whispers: "{speech}"'
            msg_location = None
```

**Comment by Griatch** [comment]
Very true, that line caused the next conditional to always be True. Thanks the report and suggestion on solution. :)

**Comment by Griatch** [comment]
Resolved in #6f3949d67a6119d2f84d9a4f55599987ac98a89e
